docs/contributing/test-components-using-accessibility-acceptance-criteria.md

```diff
@@ -0,0 +1,154 @@
+
+We recommend you test your component by putting it inside a realistic page. For example, a complete page that uses realistic page content and contains elements like a header, footer and so on.
+
+Check if the desired behaviour happens when testing. For example, whether [screen readers announce the headings](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#how-to-test). As you test, take notes about whether the component or iteration met the criteria, or if there were issues you need to investigate further.
+
+We have created a [template for capturing findings from accessibility tests](https://docs.google.com/spreadsheets/d/1O0rWoH3ah_rj2iJ2We9VfyHNOr0gOVItXRy3vKzuYSw/edit). If it could help you, download it or make your own copy. If you cannot access the spreadsheet, [contact us]( https://design-system.service.gov.uk/#support) and we will make sure you get a copy.
+
+## GOV.UK Design System - accessibility acceptance criteria
+
+Before we publish a component on the GOV.UK Design System, it should meet the following accessibility requirements.
+
+### General accessibility acceptance criteria
+
+To show clearly to users, components should:
+
+- have text with a colour contrast of at least 4.5:1 between text and the background
+- not depend on colour alone to communicate information
+- not move (for example, flash or blink)
+- be visually distinct from other elements on the page
+- use [semantic markup](https://www.w3.org/WAI/WCAG21/Techniques/general/G115.html)
+- if necessary, hide and display content consistently for both screen reader users and sighted users - [learn more about hiding and displaying content](https://webaim.org/techniques/css/invisiblecontent/#intro)
+
+To make sure users can use them, components should at least be operable with:
+
+- a keyboard
+- voice
+- touch
+- a mouse
+- the rotor or equivalent, if your screen reader has one - [learn how to use the rotor and local and global context menus](https://dequeuniversity.com/screenreaders/)
+a screen reader in browse or reader mode
+- any other modes your screen reader might have - [learn how to use different modes](https://dequeuniversity.com/screenreaders/)
+
+Behaviour may vary between different screen reader modes, which could affect what meets the definition of a component being 'operable'.
+
+To give users the information they need, components should:
+
+- contain markup in an order that makes sense to users
+- announce any changes to all users - [see our accessibility workshop slides for an example about informing users](https://www.google.com/url?q=https://docs.google.com/presentation/d/1tPocE9yr6VyE33Y_xzeV4jsHbU6z3vla3IJT1Gqwyoc/edit%23slide%3Did.ge5d201f83b_0_6&sa=D&source=editors&ust=1629108717555636&usg=AOvVaw1nrYs6mVY6yelTob5NRCzC)
+- be labelled as a [landmark region](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/main.html), if appropriate
```

Comment:
This page seems to be talking specifically about the `main` landmark, but there are other landmark regions. Can we link anywhere more generic? Maybe https://www.w3.org/TR/wai-aria-practices/examples/landmarks/index.html ?

Reply:
```suggestion
- be labelled as a [landmark region](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/index.html), if appropriate
```